docs/consume-packages/Central-Package-Management.md

```diff
@@ -9,79 +9,110 @@ ms.topic: conceptual
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
   </PropertyGroup>
   <ItemGroup>
-    <PackageVersion Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageVersion Include="PackageA" Version="1.0.0" />
+    <PackageVersion Include="PackageB" Version="2.0.0" />
   </ItemGroup>
 </Project>
 ```
 
-For each project, you then define a `<PackageReference />` but omit the `Version` attribute since the version will be obtained from a corresponding
-`<PackageVersion />` item.
+In each project file, define `<PackageReference />` elements without the `Version` attribute.
+The version will be resolved from the corresponding `<PackageVersion />` entry in `Directory.Packages.props`.
 
 ```xml
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" />
+    <PackageReference Include="PackageA" />
   </ItemGroup>
 </Project>
 ```
 
-Now you're using central package management and managing your versions in a central location!
+Now you're using Central Package Management and managing your versions in a central location!
 
-## Central Package Management rules
+### Using Different Versions for Different Target Frameworks
 
-The `Directory.Packages.props` file has a number of rules with regards to where it's located in a repository's directory and its context. For the sake of
-simplicity, only one `Directory.Packages.props` file is evaluated for a given project.
+As NuGet packages evolve, package owners may drop support for older target frameworks.
+This can cause issues for developers of libraries that still target older frameworks but want to reference newer versions of packages for newer target frameworks.
 
-What this means is that if you had multiple `Directory.Packages.props` files in your repository, the file that is closest to your project's directory will
-be evaluated for it. This allows you extra control at various levels of your repository.
+For example, if your project targets .NET Standard 2.0, .NET 8.0, and .NET Framework 4.7.2, but `PackageA` no longer supports .NET Standard 2.0 in its latest version, you can specify different versions for each target framework.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net8.0;net472</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="PackageA" />
+  </ItemGroup>
+</Project>
+```
 
-Consider the following repository structure:
+In this case, define different versions for each target framework in your `Directory.Packages.props` using [MSBuild conditions](/visualstudio/msbuild/msbuild-conditions):
+
+```xml
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="PackageA" Version="1.0.0" Condition="'$(TargetFramework)' == 'netstandard2.0'" />
```

Comment:
Do you think it's worth calling out that PM UI and just general add package supported is limited in these multi targeted scenarios?

Reply:
I don't feel strongly either way, part of me thinks its not worth calling so as not to confuse people but let me know your thoughts.